cmd/admin-handlers-users.go

```diff
@@ -679,7 +680,13 @@ func (a adminAPIHandlers) AddServiceAccount(w http.ResponseWriter, r *http.Reque
 		requestorParentUser = cred.ParentUser
 		requestorIsDerivedCredential = true
 	}
-
+	if targetUser != cred.AccessKey {
+		if _, ok := globalIAMSys.GetUser(ctx, targetUser); !ok {
```

Comment:
I don't think globalIAMSys.GetUser() works for LDAP users for example (virtual users)

Reply:
updated PR to exclude LDAP users from this check - targetUser is validated with a lookup for LDAP users further down

Reply:
> updated PR to exclude LDAP users from this check - targetUser is validated with a lookup for LDAP users further down

I think even OIDC needs to be excluded, they are also virtual users. I think this can help 'globalIAM.GetUsersSysType()'